### PR TITLE
Issue #131 : Issues Section Added.

### PR DESCRIPTION
**Pull request by aayushjoglekar**
Issue #131 resolved.

https://aayushjoglekar.github.io/OpenCode-Collaborative/  

screenshot from 2018-01-12 17-22-22  
screenshot from 2018-01-12 17-22-08

The projects haven't been included but instructions are present in index.html to create more projects.
Projects are not being fetched directly right now. I think It won't be convenient enough because the projects won't be increasing in number each day.

Hence, my suggestion would be to create an issue in which one can include all the projects rightnow.

screenshot from 2018-01-12 17-07-54  
screenshot from 2018-01-12 17-08-10

### CODE REVIEW

**Comment by aswinzz** [comment]
Good work !
@aayushjoglekar  create a comment in  "suggest new issue". we will create new issue for that.
can you add the link in the navbar too!

**Comment by aayushjoglekar** [comment]
Since a new PR will be created. This can be merged for someone to work on it.

File: copy.txt [hunk 1]
```diff
@@ -0,0 +1,138 @@
+ <section id="about">
```
Comment (Anupam-dagar) [inline copy.txt]: why is this file present?

**Comment by Anupam-dagar** [comment]
LGTM :+1:

**Pull request merged**